src/olympia/addons/migrations/0031_delete_category.py

```diff
@@ -0,0 +1,16 @@
+# Generated by Django 2.2.18 on 2021-02-09 13:26
```

Comment:
This doesn't seem to have anything to do with this patch.

Reply:
I only generated for the `ratings` app but django wanted to generate this within the `addons` app too (and then made `0002_ratingaggregate.py` depend on it), so I'd either need to manually tinker with them or land it beforehand in a separate PR.  I figured the migration needed generating and committing at _some point_ (after the 2021-02-11 tag) anyway so 🤷 
What do you think?